packages/next-dashboard/src/components/layout/Content.js

```diff
@@ -14,10 +16,12 @@ type Props = {
 
 const Content = ({ children, contentContainerWidth, header }: Props) => (
   <div
+    id="dashboard-content"
```

Comment:
```suggestion
```

Reply:
Oh yeah, forgot to comment about that, I was wondering what the ID is used for too